app/api/dao/mentorship_relation.py

```diff
@@ -139,8 +139,7 @@ def list_mentorship_relations(user_id=None, accepted=None, pending=None, complet
         all_relations = user.mentor_relations + user.mentee_relations
 
         # add extra field for api response
-        for relation in all_relations:
-            setattr(relation, 'sent_by_me', relation.action_user_id == user_id)
+        map((lambda relation: setattr(relation, 'sent_by_me', relation.action_user_id == user_id), all_relations))
```

Comment:
I think map function return mapped array which you need to assign somewhere. Using setter inside of map is kind of breaking the purpose of mapper I would recommend to use loop lamdas or creating new objects and reasign.